docs/DeveloperGuide.md

```diff
@@ -314,6 +314,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+**Use case: Install the app**
+MSS 
+1. User execute the installation package
+2. The app ask for continue or cancel with a pop up window
+3. User click continue
+4. The app install the app
+5. The app show a pop up window to inform user the installation is successful
```

Comment:
Perhaps the Use Case can be reformatted to remove UI details. I feel that point 4 is not very clear.